code/framework/extension/src/main/java/io/cattle/platform/extension/impl/ExtensionManagerImpl.java

```diff
@@ -61,6 +51,21 @@
     }
 
     @Override
+    public Map<String, Object> map(String key) {
+        return getExtensionMapInternal(key);
+    }
+
+    protected ExtensionMap<String, Object> getExtensionMapInternal(String key) {
+        ExtensionMap<String, Object> map = extensionMaps.get(key);
+        if (map == null) {
+            Map<String, Object> inner = started ? getExtensionMapInternal(key) : Collections.<String, Object>emptyMap();
```

Comment:
the call to getExtensionMapInternal(key) looks wrong to me here.  did you mean to call getMap(key)?